ansible/group_vars/nimbus.mainnet.yml

```diff
@@ -6,42 +6,118 @@ bootstrap__extra_volume_bind_path: null
 # SWAP
 swap_file_size_mb: 2048
 
-# Go-Ethereum
-geth_service_name: 'geth-{{ geth_network_name }}'
+# Ports used for all 3 EL nodes as they are mutually exclusive.
+exec_layer_p2p_port:     '{{ 30303 + 3*idx|int + 1 }}'
+exec_layer_rpc_port:     '{{ 8545  +   idx|int + 1 }}'
+exec_layer_authrpc_port: '{{ 8551  +   idx|int + 1 }}'
+exec_layer_metrics_port: '{{ 6060  +   idx|int + 1 }}'
```

Comment:
I would put this at the top, below swap but above Geth settings, since it's not just for Geth.